### PR TITLE
[T4A4][T13-B2] Gabriel Gheorghian

### PR DESCRIPTION
**Pull request by GabrielGhe**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address implements Printable {
 
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/Address.java:12]: why public ?

File: src/seedu/addressbook/data/person/ReadOnlyPerson.java [hunk 2]
```diff
@@ -82,4 +82,13 @@ default String getAsTextHidePrivate() {
...
+    
+    default String getPrintableString(Printable... printables) {
+        final StringBuilder builder = new StringBuilder();
+        for (Printable printable : printables) {
+            builder.append(printable.getPrintableString());
+            builder.append(" ");
+        }
+        return builder.toString();
+    }
 }
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/ReadOnlyPerson.java:94]: Good place to use getPrintableString(Printable... printables). Nice work

**Comment by KulaniT** [comment]
@GabrielGhe : Nice work implementing ISP. I see you can improve your code little more if pay attention on which access modifiers suitable. Overall good work. Keep it up. :)

Please close the PR after reading comments

File: src/seedu/addressbook/data/person/Address.java (hunk 1 above)
Comment (GabrielGhe) [reply src/seedu/addressbook/data/person/Address.java:12]: In case they need to be used during testing

**Pull request closed without merging**